inst/schema/DownloadSubmitRequest.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "note": {
+      "type": "object",
+      "properties": {
+        "name": { "type": "string" },
+        "updated": { "type": "string" },
```

Comment:
Is this meant to be the updated timestamp for the note or the project/version? We have the latter, not the former.

Reply:
Yeah. Was thinking last updated time, where the last updated time for the project is the the latest time from all the different versions. Assuming that is what we show in the UI? I am thinking this should just match the times we show in the UI
![Screenshot_20220519_150500](https://user-images.githubusercontent.com/39248272/169312986-4b7ae343-525f-4080-a78b-bd2736cf4eb8.png)

I meant to ask - what format should we return here? Is a string sensible?